FirebaseVertexAI/Sample/FunctionCallingSample/Screens/FunctionCallingScreen.swift

```diff
@@ -65,6 +65,9 @@ struct FunctionCallingScreen: View {
             }
           }
         })
+        .onTapGesture {
+          focusedField = nil
+        }
```

Comment:
Thanks, I like how this closes the keyboard if you tap outside the text field!

Reply:
🙏